content/docs/09.5-security/11-product-architecture/05-tenant-cluster.md

```diff
@@ -0,0 +1,97 @@
+
+Each tenant cluster has a management agent that runs as a pod. This agent has an outbound internet connection to Palette using static Network Address Translation (NAT) with Transport Layer Security (TLS) protocol v1.2 or higher and a hardened cipher suite. The agent periodically reports health, heartbeat, and statistics and connects to Palette's public repository over HTTPS for any out-of-the-box integration packs.
+
+In a self-hosted environment, where Palette is typically deployed on-prem behind a firewall, you must ensure security controls in your environment. Palette automatically generates security keys at installation and stores them in the management cluster. You can import an optional certificate and private key to match the management cluster Fully Qualified Domain Name (FQDN). Palette supports enabling disk encryption policies for management cluster VMs if required.
+
+
+## Hardened Operating System
+
+Palette provides Ubuntu or CentOS images for supported cloud environments. Images that are hardened according to Center for Internet Security (CIS) Benchmarks are used to launch control planes and worker nodes for the Kubernetes cluster.
+
+Palette's OS hardening utility performs the following tasks:
+
+<br />
+
+- Applies the latest available security updates.
+
+
+- Hardens SSH server parameters, network parameters (sysctl), and system files by ensuring proper file permissions are set.
+
+
+- Removes legacy services and graphical user interface (GUI) packages.
+
+Palette allows you to set up OS patching policies. You can patch the base OS when you deploy the cluster. Refer to the [OS Patching](/clusters/cluster-management/os-patching) documentation to learn more.
+
+<br />
+
+## Hardened Containers
+
+Container images for various application services are built using distroless images, which have significantly fewer packages and improve security by reducing attack surface.
+
+All container images are scanned for vulnerability using Palo Alto Networks Prisma Cloud (Twistlock) Defender before being published to a repository or deployed to the SaaS platform.
+
+<br />
+
+## Hardened Kubernetes
+
+Spectro Cloud has a fully automated Kubernetes verification system that adds the newest patch version of Kubernetes to its public repository. All Kubernetes packs are hardened according to CIS Benchmarks.
+
+We assess major Kubernetes versions based on the extent of changes.
+
+Kubernetes run-time security support is achieved through a variety of add-on packages, such as Sysdig Falco and Twistlock.
+
+Kubernetes cluster authentication can be optionally integrated with Kubelogin using OpenID Connect (OIDC) authentication and authorization against an external Identity Provider (IDP).
```

Comment:
@cloudmaniac Line 81 repeats line 89. Maybe delete line 81 and the heading in line 87 and have just one section under "Hardened Kubernetes". 

Or just delete line 81.

Reply:
OK, done